ggml-backend-impl.h

```diff
@@ -103,6 +103,11 @@ extern "C" {
         // check if the backend supports an operation
         bool (*GGML_CALL supports_op)(ggml_backend_t backend, const struct ggml_tensor * op);
 
+        // check if the backend want to run an operation, even if the weights are allocated in a CPU buffer
+        // these should be expensive operations with large batch sizes that may benefit from running on this backend
```

Comment:
```suggestion
        // these should be compute-bound operations (e.g. large batch matrix multiplication) for which the overhead of data transfers from/to CPU is offset by a reduction in operation runtime
```

Reply:
That's not really the way it is used, any operation with batch size > 32 will be offloaded. This is because there is no advantage to moving the activations back to the CPU to run cheap operations, you can keep them on the GPU and run everything on the device. Even for 1D weights, it is cheaper to copy the weight to the GPU than the activations to the CPU. That's the source of the performance improvement here.